### ISSUE TITLE
HTTP ERROR 403 when User-Agent=Mozilla in request header

### ISSUE THREAD

**Issue by aprii** [Cognifide/aem-stubs#28]
**Steps to reproduce:**
Send any POST request to AEM Stubs. The request has to contain the following header:
`User-Agent=Mozilla`

**Actual results:**
403 Forbidden
```
<html>
<head>
<meta http-equiv="Content-Type" content="text/html;charset=ISO-8859-1"/>
<title>Error 403 </title>
</head>
<body>
<h2>HTTP ERROR: 403</h2>
<p>Problem accessing /stubs/update-interaction. Reason:
<pre>    Forbidden</pre></p>
<hr /><i><small>Powered by Jetty://</small></i>
</body>
</html>
```

**Expected results:**
It should be forwarded to Wiremock engine.

**Comment by pun-ky** [comment]
it is because the empty referrer

for example, enable empty via OSGi config or whitelist user-agent to bypass it:

![image](https://user-images.githubusercontent.com/10673561/83064937-ea844e80-a062-11ea-9e0c-cc79c8b8b00d.png)

**Comment by pun-ky** [comment]
also implemented a bypass which effectively disables all possible request filters registered on AEM that could make stubs not working https://github.com/Cognifide/aem-stubs/pull/29

@aprii @marcinkp so probably no additional configuration will be required

TODO only test regression of that fix on AEM 6.3 & 6.4

**Comment by aprii** [comment]
Thanks @pun-ky for quick response

**Comment by pun-ky** [comment]
fixed by #29 ; released in 1.0.4
